demos/common/src/ie_config_helper.cpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "samples/ie_config_helper.hpp"
+#include "samples/args_helper.hpp"
+
+#include <cldnn/cldnn_config.hpp>
+#include <vpu/myriad_config.hpp>
+
+void formatDeviceString(std::string& deviceString) {
+    bool preserveCase = false;
+
+    for (size_t i = 0; i < deviceString.size(); ++i) {
+        // These two conditions handle the special case of MYRIAD device names in "ma1234" format, where letters should
+        // not be transformed to upper case.
+        if (deviceString[i] == 'm' && i+1 != deviceString.size() && deviceString[i+1] == 'a') {
+            preserveCase = true;
+        }
+        if (preserveCase == true && deviceString[i] == ',') {
+            preserveCase = false;
+        }
+
+        if (!preserveCase) {
+            deviceString[i] = std::toupper(deviceString[i]);
+        }
+    }
+}
+
+std::map<std::string, std::string> createSimpleConfig(const std::string& deviceString) {
```

Comment:
The idea I had behind introducing the simple config is that demos will use it.

This config was meant to be used with demos that have only one stream. `CONFIG_VALUE(CPU_THROUGHPUT_AUTO)` may be greater than 1. For CPU and GPU the default value is already 1, it may be better to add `{{InferenceEngine::MYRIAD_THROUGHPUT_STREAMS, "1"}}` to network loading unconditionally in each one stream demo.

Reply:
"one stream demo" is a demo in which the user can't specify the number of streams on startup? Therefore, everything except for classification, object detection ssd and security barrier is considered "one stream demo"?

Reply:
Seems to be so, but may be there are other demos utilizing more than one InferRequest for a given net and having default(2) number of streams for VPU can be beneficial for them.